src/Sql/Sql/Database/Cmdlet/RevertAzureSqlDatabaseTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Azure.Commands.Sql.Database.Model;
+using System.Management.Automation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet for Revert Azure Sql Database encryption protector
+    /// </summary>
+    [Cmdlet("Invoke", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseTransparentDataEncryptionProtectorRevert", SupportsShouldProcess = true), OutputType(typeof(AzureSqlDatabaseModel))]
+    public class RevertAzureSqlDatabaseTransparentDataEncryptionProtector : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseModel>
```

Comment:
Change the class name and file name to match invoke-AzSqlDatabaseTransparentDataEncryptionProtectorRevert